code/src/Header.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
```

Comment:
Empty component?

Reply:
I have put my image in the header, so if I remove that component. Will the picture not disappear then?